fileio.c

```diff
@@ -0,0 +1,37 @@
+//Matthew Kellerman - gets numbers from input file and finds the average and the squares the numbers. It then prints it to an output file
```

Comment:
your code mostly works, just a few things!
1. You printed the last number twice, which then made your average off by a little.
2. you left 7 warnings, mostly because of array accesses out of bounds!